triplestore-jena-memory/src/test/resources/init_ctms_tdb/CTMS-SK-NSC-20221901.ttl

```diff
@@ -0,0 +1,3359 @@
+<http://data.europa.eu/esco/ctms/mapping/3b5f555b-21e9-4a5e-b8ea-41d7e7ce7750>
```

Comment:
not sure if we should put customer data in opensource?

Reply:
uh, yes, it's definitely a mistake, I'll replace, apology

Reply:
buuut, as a second thought, this will be published along with esco data

Reply:
I understand. But maybe we should keep a2 a bit customer neutral?